deploy/docker/fs/opt/appsmith/utils/bin/move-to-postgres.mjs

```
@@ -147,16 +147,39 @@ function replacer(key, value) {
  * Updates:
  * 1. Changes the _id field to id, and removes the _id field.
+ * 2. Replaces the _class field with the appropriate type field.
  * @param {Document} obj - The object to transform.
  * @returns {void} - No return value.
  */
 function transformFields(obj) {
   for (const key in obj) {
-    if (key === "_id") {  // Change the _id field to id
+    if (key === "_id") {
       obj.id = obj._id.toString();
       delete obj._id;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)